client/debug/main.go

```diff
@@ -257,3 +258,20 @@ $ %s debug raw-bytes [72 101 108 108 111 44 32 112 108 97 121 103 114 111 117 11
 		},
 	}
 }
+
+func PrefixesCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "prefixes",
+		Short: "List prefixes used for Human-Readable Part (HRP) in Bech32",
+		Long: fmt.Sprintf(`List prefixes used in Bech32 addresses.
+
+Example:
+$ %s debug prefixes
+			`, version.AppName),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.Printf("Bech32 Acc: %s\n", sdk.GetConfig().GetBech32AccountAddrPrefix())
```

Comment:
We should print all of them too.

Reply:
Do you want to print the _PubPrefixes_ too?

Reply:
Also, this raises the question - why `simd debug addr` doesn't print all of them?

Reply:
I don't have a good reason, but we should print the address, validator and consensus prefixes.